apps/src/lib/kits/maker/ui/SetupChecklist.jsx

```diff
@@ -15,21 +15,26 @@ import {
   isLinux
 } from '../util/browserChecks';
 import ValidationStep, {Status} from '../../../ui/ValidationStep';
+import {BOARD_TYPE} from '../CircuitPlaygroundBoard';
+import experiments from '../../../../util/experiments';
```

Comment:
Tiny style nit, use the `@cdo/apps` alias for deeply-nested dependencies:
```suggestion
import experiments from '@cdo/apps/util/experiments';
```